### PR TITLE
CIV-5797 - Transform Mediation model - CCD

### PR DESCRIPTION
**Pull request by portizdev**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5797

### Change description ###
Transform Mediation Model into CCD Model

**Framework developed by @kenneth-hmcts .**

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by LeonardoPalmeiro — DISMISSED** [review]

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json [hunk 1]
```diff
@@ -17,5 +17,25 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "respondent1ResponseCui",
```
Comment (TatsyMaslova) [inline ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json:23]: I am not sure that this is a good name for a field. i dont think ccd should know which project/application is populating this field. In this case this is a lip response. Perhaps it should be respondent1LipResponse?

File: ccd-definition/ComplexTypes/Mediation-CUI-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "ID": "MediationFromCui",
```
Comment (TatsyMaslova) [inline ccd-definition/ComplexTypes/Mediation-CUI-nonprod.json:3]: Again could be LipMediation

File: ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json (hunk 1 above)
Comment (portizdev) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json:23]: The structure of the model was @kenneth-hmcts 's idea. Regarding this he commented me that we should reference that the fields come from CUI.

Comment (PaulH-HMCTS) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json:23]: I agree with Tanya - model should always be separate from the view impl

Comment (portizdev) [reply ccd-definition/AuthorisationCaseField/AuthorisationCaseField-CUI-nonprod.json]: Solve, please review it again <3

File: ccd-definition/ComplexTypes/Mediation-CUI-nonprod.json (hunk 2 above)
Comment (portizdev) [reply ccd-definition/ComplexTypes/Mediation-CUI-nonprod.json]: Solve, please review it again <3

**Review by TatsyMaslova — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by nishm — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by nishm — APPROVED** [review]

**Pull request merged**